src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -127,6 +127,10 @@ function statusMessageRowsRenderer(instance, td, row) {
     });
 }
 
+enrollHandsontable.addHook('beforeColumnSort', () => {
+    updateEnrollHandsontableCellSettings(resetDefaultViewRenderer);
```

Comment:
Not sure what exactly this commit does from the description.

Reply:
After a column sort, any status message on the spreadsheet would be not be in the same position anymore, so this hook serves to reset the cell styles/tool tips to default just before a column sort.